src/main/java/com/creavispace/project/entity/ProjectReplyEntity.java

```diff
@@ -0,0 +1,48 @@
+package com.creavispace.project.entity;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import lombok.Getter;
+
+@Getter
+@Entity
+@Table(name = "project_reply")
+public class ProjectReplyEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne
+    @JoinColumn(name = "parent_reply_id", columnDefinition = "BIGINT COMMENT '이전댓글ID'")
```

Comment:
` columnDefinition = "BIGINT COMMENT '이전댓글ID'" ` 이렇게 하면 DB에 어떤 결과로 들어가나요? COMMENT에 대해 제가 잘 몰라서요

Reply:
```suggestion
    @Column(name = "kind" , nullable = false, columnDefinition = "VARCHAR(20) COMMENT 'PERSONAL OR TEAM'")
    private String kind;
```
ProjectEntity의 kind컬럼을 예를 들면 DB 타입이 VARCHAR(20)으로 저장되고 COMMENT란에 PERSONAL OR TEAM으로 기입됩니다.
아래 사진은 MySQL Wrokbench에 저장된 project Table의 kind 컬럼 데이터입니다.

![image](https://github.com/CreaviSpace/CreaviSpace-Backend/assets/82141580/70e6df7f-dadc-46a7-9924-d54f660cd6fb)